pkg/config/config.go

```diff
@@ -83,6 +103,20 @@ func NewDefaultConfig() *Config {
 		ManagerOptions: ManagerOptions{
 			LeaderElectionNamespace: leaderElectionNamespaceDefault,
 		},
+		Controllers: Controllers{
+			Build: ControllerOptions{
+				MaxConcurrentReconciles: 1,
+			},
+			BuildRun: ControllerOptions{
+				MaxConcurrentReconciles: 1,
+			},
+			BuildStrategy: ControllerOptions{
+				MaxConcurrentReconciles: 1,
+			},
+			ClusterBuildStrategy: ControllerOptions{
+				MaxConcurrentReconciles: 1,
+			},
+		},
```

Comment:
A quick question, is that possible to skip this default setting here? And let framework to set default values for us, like before.

We manually set default to 1 for all, but I am not sure if the framework change the default value in future, then they will be mismatch.

Reply:
Done.

Reply:
It looks better to have a check by using `zero`.

Thanks Sascha!